spec/features/static_pages_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+describe "Home", type: :feature  do
+  let!(:card) { Card.create(original_text: "example",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.